code-samples/calendar/python/calendar.py

```diff
@@ -0,0 +1,28 @@
+import os
+from nylas import Client
+from dotenv import load_dotenv
+
+load_dotenv()
+
+nylas = Client(
+    api_key = os.getenv("API_KEY")
+)
+
+#creating a calendar
+calendar = {
+    "name": "New Calendar Name"
+}
+
+#printing calendars
+no_calendar= 0
+print("SNo. | Calendar Name | Calendar ID\n")
+for i in calendar:
+    print(f"{no_calendar}. {i.name} | {i.id}")
+    no_calendar+=1
+
+try:
+    new_calendar = nylas.calendars.create(calendar)
```

Comment:
I don't see the use of a grant id here.

Let me know if you are able to test this using the v3 dashboard. Because the grant ID is required.

As an example, here is how I would list available calendars with the grant id as the identifier.
```
response = nylas.calendars.list(identifier=os.getenv("GRANT_ID"))
calendars = response.data 
```

Reply:
@JuanitaCathy - this code needs to be updated to accept grant ID - view above example provided.

Here is the reference code sample: https://github.com/nylas-samples/nylas-hacktoberfest-2023/tree/main/code-samples/calendar-event/python/delete_calendar_event